src/components/common/BackButton/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { useRouter } from 'next/router';
+import LeftArrow from '@/assets/svgs/left-arrow.svg';
+
+interface BackButton {
+  href?: string;
+}
+
+export default function BackButton({ href }: BackButton) {
+  const router = useRouter();
+
+  function handleBack() {
+    router.back();
+  }
+
+  function handleClick() {
+    href && router.push(href);
+  }
```

Comment:
💊 이거는 `handleClick` 하나로 처리할 수 있을 거 같아요~

밑에는 참고용입니다!
```suggestion
  function handleClick() {
    if (href) {
      router.push(href);
      return;
    }
    router.back();
  }
```

Reply:
반영했습니다 감사합니다!